src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -0,0 +1,226 @@
+        {
+            Caption = 'No. of tests running';
+            trigger OnValidate()
+            var
+                AITTestMethodLine: Record "AIT Test Method Line";
+                AITTestSuiteMgt: Codeunit "AIT Test Suite Mgt.";
+            begin
+                if "No. of Tests running" < 0 then
+                    "No. of Tests running" := 0;
+
+                if "No. of Tests running" <> 0 then
+                    exit;
+
+                case Status of
+                    Status::Running:
+                        begin
+                            AITTestMethodLine.SetRange("Test Suite Code", "Code");
+                            AITTestMethodLine.SetRange(Status, AITTestMethodLine.Status::" ");
+                            if not AITTestMethodLine.IsEmpty then
+                                exit;
+                            AITTestSuiteMgt.SetRunStatus(Rec, Rec.Status::Completed);
+                            AITTestMethodLine.SetRange("Test Suite Code", "Code");
+                            AITTestMethodLine.SetRange(Status);
+                            AITTestMethodLine.ModifyAll(Status, AITTestMethodLine.Status::Completed, true);
+                        end;
+                    Status::Cancelled:
+                        begin
+                            AITTestMethodLine.SetRange("Test Suite Code", "Code");
+                            AITTestMethodLine.ModifyAll(Status, AITTestMethodLine.Status::Cancelled, true);
+                        end;
+                end;
+            end;
+        }
+        field(11; Tag; Text[20])
+        {
+            Caption = 'Tag';
+            DataClassification = CustomerContent;
+        }
+#pragma warning disable AA0232
+        field(12; "Total Duration (ms)"; Integer)
+#pragma warning restore AA0232
+        {
+            Caption = 'Total Duration (ms)';
+            ToolTip = 'Specifies the Total Duration (ms) for executing all the tests in the current version.';
+            Editable = false;
+            FieldClass = FlowField;
+            CalcFormula = sum("AIT Log Entry"."Duration (ms)" where("Test Suite Code" = field("Code"), Version = field("Version"), Operation = const('Run Procedure'), "Procedure Name" = filter(<> '')));
+        }
+        field(13; Version; Integer)
+        {
+            Caption = 'Version';
+            Editable = false;
+        }
+        field(16; "Base Version"; Integer)
+        {
+            Caption = 'Base Version';
+            DataClassification = CustomerContent;
+            MinValue = 0;
+            trigger OnValidate()
+            begin
+                if "Base Version" > Version then
+                    Error(BaseVersionMustBeLessThanVersionErr)
+            end;
+        }
+        field(19; RunID; Guid)
+        {
+            Caption = 'Unique RunID';
+            Editable = false;
+        }
+        field(20; ModelVersion; Option)
+        {
+            Caption = 'AOAI Model Version';
```

Comment:
AOAI here also and ModelVersion field name, i would add spacing

Reply:
Done, let's keep the caption as it is for now.